src/partition/repart.c

```diff
@@ -6810,6 +6812,9 @@ static int help(void) {
                "                          signatures\n"
                "     --certificate=PATH   PEM certificate to use when generating verity\n"
                "                          roothash signatures\n"
+               "     --signing-engine=ENGINE\n"
+               "                          Specify OpenSSL engine to use when generating verity\n"
```

Comment:
engine/provider

Reply:
here too, say "private-key" is usually a path, but in some cases might be something else.

maybe use the wording from th eukify man page